webrtc.html

```diff
@@ -528,23 +520,60 @@
           </li>
 
           <li>
-            <p>Create a <code><a>MediaStream</a></code> object
-            <var>stream</var>, to represent the incoming media stream.</p>
-          </li>
+            <p>Run the following steps to create a track
+            representing the incoming component:</p>
 
-          <li>
-            <p>Run the <a href="#represent-component-with-track">algorithm</a>
-            to represent an incoming component with a track for each incoming
-            component.</p>
+            <ol>
+              <li>
+                <p>Create a <code><a>MediaStreamTrack</a></code> object
+                <var>track</var> to represent the component.</p>
+              </li>
+
+              <li>
+                <p>Initialize <var>track's</var> <code><a href=
+                "getusermedia.html#dom-mediastreamtrack-kind">kind</a></code>
+                attribute to "<code>audio</code>" or "<code>video</code>"
+                depending on the media type of the incoming component.</p>
+              </li>
+
+              <li>
+                <p>Initialize <var>track's</var> <code><a href=
+                "getusermedia.html#dom-mediastreamtrack-id">id</a></code>
+                attribute to the component track id.</p>
+              </li>
+
+              <li>
+                <p>Initialize <var>track's</var> <code><a href=
+                "getusermedia.html#dom-mediastreamtrack-label">label</a></code>
+                attribute to "<code>remote audio</code>" or "<code>remote
+                video</code>" depending on the media type of the incoming
+                component.</p>
+              </li>
+
+              <li>
+                <p>Initialize <var>track's</var> <code><a href=
+                "getusermedia.html#dom-mediastreamtrack-readystate">readyState</a></code>
+                attribute to <code>muted</code>.</p>
+              </li>
+
+            </ol>
+          </li></p>
 
             <p class="note">The creation of new incoming
-            <code>MediaStream</code>s may be triggered either by SDP
-            negotiation or by the receipt of media on a given flow. 
+            <code>MediaStreamTrack</code>s may be triggered either by SDP
+            negotiation or by the receipt of media on a given flow.
             <!--  [[OPEN ISSUE: How many <code>MediaStream</code>s are created
                 when you receive multiple conflicting pranswers?]] --></p>
           </li>
 
           <li>
+            Add <var>track</var> to the <code>MediaStream</code>(s) <var>streams</var>
```

Comment:
The variable "streams" turns up out of the blue here and without a proper description.